src/tasks/tasks.py

```diff
@@ -434,6 +434,45 @@ def load_data(self, path, max_seq_len):
 @register_task('cola_npi_negdet', rel_path='CoLA/npi_negdet')
 @register_task('cola_npi_cond', rel_path='CoLA/npi_cond')
 @register_task('cola_npi_adv', rel_path='CoLA/npi_adv')
+class NPITask(SingleClassificationTask):
+    '''Class for NPI-related task; same with Warstdadt acceptability task but outputs labels for test-set'''
```

Comment:
=> CoLA?

Reply:
Especially since we're using the CoLA directory and cola_ in the task names.

Reply:
This is actually another point that I've raised a couple of times... I think it's kind of confusing to put these in the CoLA folder and also have "cola" in the task name... I have suggested put them in another folder (maybe called "NPI") and also just use the folder structure in the data_generation folder. 

But if we're sticking to the current folder/task arrangement, maybe I can call this something like CoLANPITask?